### DISCUSSION TITLE
After how much time does the badge appear on Profile?

### DISCUSSION THREAD

**Discussion by JordyHers** [community/community#20864] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey guys, I was wondering how long it takes before the badge appears on your profile page.

**Comment by AnuraghSarkar** [comment, marked as answer]
It would be shown immediately after the criteria are met!

**Comment by geraldokereke** [comment]
The badge appears on your profile, immediately after the requirements of the badge are met.
Kindly see this: https://github.com/Schweinepriester/github-profile-achievements

**Comment by aaronlamz** [comment]
The badge will be displayed immediately when you complete the corresponding achievement.
more detail here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by foxyseta** [comment]
It is displayed immediately upon achievement.

If your profile page was already opened in another browser tab, though, you'll need to refresh it.
If you still do not see your badge, then you didn't actually meet all of the requirements.

**Comment by reyhanatash** [comment]
I merged a pull request before 5 minute but 
you didn't get me Quickdraw badge  :((

based on https://github.com/drknzz/GitHub-Achievements
